Amq_List_Broker-main/activemq.py

```diff
@@ -0,0 +1,156 @@
+import  boto3
+import requests
+import json
+import os
+from botocore.exceptions import ClientError
+import xml.etree.ElementTree as ET
+
+client_Name = os.environ['client_Name']
```

Comment:
variable names here in snakecase to be consistent.. client_Name=> client_name, mq_user_Name=> mq_user_name

Reply:
done